app/controllers/main/routes.py

```diff
@@ -125,11 +127,46 @@ def viewUsersProfile(username):
                 volunteer = volunteer,
                 backgroundTypes = backgroundTypes,
                 allBackgroundHistory = allBackgroundHistory,
-                currentDateTime = datetime.datetime.now()
-
+                currentDateTime = datetime.datetime.now(),
+                profileNotes = profileNotes
             )
     abort(403)
 
+@main_bp.route('/profile/addNote', methods=['POST'])
+def addNote():
+    """
+    This function adds a note to the user's profile.
+    """
+    postData = request.form
+    try:
```

Comment:
Can replace the whole first try and except here with a check for Bonner in postData since the value only ever exists when it is "on".